### DISCUSSION TITLE
Is there a way to not use span element in note syncing

### DISCUSSION THREAD

**Discussion by ulahcherubim** [windingwind/zotero-better-notes#954] [General]
Hi, I just wonder if it is possible to disable span element used for highlighting Zotero Notes native citation and Zotero Better Notes links. I want my markdown files to be plain as possible.

I think this might have to do with how markdown notes synced back to Zotero, since Zotero Notes are not markdown, they might need these elements in order to render correctly. But I wonder if is it possible to limit the span element to Zotero Notes inline citations. It make editing hard in Obsidian when Zotero Better Notes links are also rendered with html tags.

Edit: Sorry for pinging @windingwind but I couldn't figure out why.

**Comment by helsingi** [comment]
Are you talking about the content of the exported notes being underlined and blue (hyperlinked)? I would love also if the content is plain text and only the intext citation is hyperlinked back to zotero
